modules/rosa-imds-machine-pools-cli.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-imds-machine-pools-cli_{context}"]
+= Enabling Instance Metadata Service on machine pools in CLI
+
+You can select your Instance Metadata Service (IMDS) type when creating your cluster in your ROSA CLI. You can select to use both IMDSv1 and IMDSv2, or you can specify to only use IMDSv2.
+
+.Prerequisites
+
+* You installed and configured the latest AWS (`aws`), ROSA (`rosa`), and OpenShift (`oc`) CLIs on your workstation.
+* You logged in to your Red Hat account by using the `rosa` CLI.
+* You have the permissions to create and manage clusters.
+
+.Procedure
+
+. In your terminal, create a ROSA cluster with your specifications by running the following command:
++
+[source,terminal]
+----
+$ rosa create cluster --cluster <name_of_cluster> --ec2-metadata-http-tokens <required_or_optional> <1>
+----
++
+<1> You can provide a value for the `--ec2-metadata-http-tokens` flag, either `required` to enable only IMDSv2 type, or `optional` for a combination of IMDSv1 and IMDSv2. If you do not include this flag, you need to select your IMDS type during the cluster creation prompts.
++
+During the cluster creation process, you see the following line that allows you to specify if you use IMDSv2 only or if you use both IMDSv1 and IMDSv2. If you provided a value when creating your cluster, you need to confirm the selection:
```

Comment:
In context of the second sentence you added, here is a nit/suggestion:

```suggestion
. Confirm your selection:
```

I'm suggesting making this a step now that I see that it's something the user needs to do, sorry for contradicting my suggestion from the first review. I'm also trimming the context down in the suggestion since I feel like it's implied that the user followed the previous step as instructed, but that's more of a small nit.

Reply:
+1 on making this a step again, and also simplifying the instruction with the assumption they ran the previous step w/ the flag. Possibly a little more context could work, but the even simpler version seems fine too:

```
After running the command, confirm your selection at the following line:
```

Also reminder to remove the `+` continuation on line 27.